app/smc/pytest/e2e_smoke.py

```diff
@@ -188,3 +189,25 @@ def test_fw_bundle_version(arc_chip):
         telemetry.fw_bundle_version == exp_bundle_version
     ), f"Firmware bundle version mismatch: {telemetry.fw_bundle_version:#010x} != {exp_bundle_version:#010x}"
     logger.info(f"FW bundle version: {telemetry.fw_bundle_version:#010x}")
+
+
+@pytest.mark.reset
+def test_reset_accessible(arc_chip):
+    """
+    Checks that the chip is properly accessible after tt-smi resets
+    """
+    smi_reset_cmd = "tt-smi -r"
+    total_tries = 10
+    fail_count = 0
+    for i in range(total_tries):
+        logger.info(f"Iteration {i}:")
+        smi_reset_result = subprocess.run(
+            smi_reset_cmd.split(), capture_output=True, check=False
+        ).returncode
+        logger.info(f"'tt-smi -r' returncode:{smi_reset_result}")
+
+        if smi_reset_result != 0:
+            fail_count += 1
+
+    logger.info(f"'tt-smi -r' failed {fail_count}/{total_tries} times.")
+    assert fail_count <= 1, "'tt-smi -r' fail count greater than expected threshold"
```

Comment:
We should probably require that 10 resets are successful. For longer duration tests where we reset e.g. 100-300 times, I think we can probably settle for 95% or better success rate.
```suggestion
    assert fail_count == 0, "'tt-smi -r' fail count greater than expected threshold"
```